msal/src/main/java/com/microsoft/identity/client/internal/controllers/LocalMSALController.java

```diff
@@ -247,4 +256,24 @@ public AcquireTokenResult acquireTokenSilent(
 
         return acquireTokenSilentResult;
     }
+
+    public void removeLocalAccount(@Nullable final IAccount account,
+                                   @NonNull final PublicClientApplicationConfiguration configuration,
+                                   @NonNull final PublicClientApplication.AccountsRemovedCallback callback) {
+        // FEATURE SWITCH: Set to false to allow deleting Accounts in a tenant-specific way.
+        final boolean deleteAccountsInAllTenants = true;
```

Comment:
This looks little confusing. We are trying to remove one account and making use of variable with deleteAccountsInAllTenants.

Reply:
In this case, we wish to remove both home and any guest accounts so we _are_ removing multiple accounts.

Because a user principal maps to multiple `AccountRecords` in guest scenarios, we (internally) support deleting `IAccounts` in multiple ways:
- Tenant-specific
   * Ex: a user is logged in Contoso and Fabrikam, but only wishes to remove the `AccountRecord` for Contoso
- Tenant-agnostic
    * Ex: a user is logged in Contoso and Fabrikam and wishes to remove both records

To make this more clear, let's rename this variable to something like `deleteHomeAndGuestAccounts` @heidijinxujia